src/actor/render.rs

```diff
@@ -45,15 +45,31 @@ impl RenderActor {
         res
     }
 
-    pub fn run(mut self) {
+    pub fn run(self) {
```

Comment:
We may rename function `run` to `spawn` and rename function `run_impl` to `run`.